084_1_kaimMatrica.cpp

```diff
@@ -0,0 +1,5 @@
+int n;                    // viršūnių skaičius
+bool briauna[MAXN][MAXN]; // briauna[i][j] reiškia, kad grafe yra briauna tarp viršūnių i ir j
+
+// Vertėjo pastaba: pascal kalbos kode grafas pateikiamas kaip struktūra,

```

Comment:
```suggestion
// Pastaba: originaliame pavyzdyje grafas pateikiamas kaip struktūra,

```

Tas pats kitose vietose.